### PR TITLE
Add readme for notebook examples

### PR DESCRIPTION
**Pull request by Pahandrovich**

### CODE REVIEW

File: examples/notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Intel(R) Extension for Scikit-learn* notebooks :snake:
+
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for known small datasets.  
+You need jupyter notebook to run these .ipynb files:
+
+```bash
+conda install -c conda-forge notebook
```
Comment (PetrovKP) [inline examples/notebooks/README.md:7]: And sklearnex

File: examples/notebooks/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Intel(R) Extension for Scikit-learn* notebooks :snake:
+
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for known small datasets.
```
Comment (PetrovKP) [inline examples/notebooks/README.md:3]:
```suggestion
This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.  
```

File: examples/notebooks/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for known small datasets.  
+You need jupyter notebook to run these .ipynb files:
+
+```bash
+conda install -c conda-forge notebook
+```  
+or  
+```bash
+pip install notebook
+```
```
Comment (PetrovKP) [inline examples/notebooks/README.md:12]: I think cool to add table with current examples and small description

**Comment by SmirnovEgorRu** [comment]
I think we need to have a link from the main README to the samples

File: examples/notebooks/README.md (hunk 3 above)
Comment (PetrovKP) [reply examples/notebooks/README.md:12]: For example
| **Algorithm** | **Workload** | **Notebook**|
|----------------|-----------------|----------------|
| **Logistic Regression** |  CIFAR-100 | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb) |
| **SVC**                         |  Adult         |     [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/svc_adult.ipynb) |
| ....                            | ....               | ....                |

File: README.md [hunk 4]
```diff
@@ -34,6 +34,7 @@ We publish blogs on Medium, so [follow us](https://medium.com/intel-analytics-so
 - [Building from Sources](https://github.com/intel/scikit-learn-intelex/blob/master/INSTALL.md)
 - [About Intel(R) oneAPI Data Analytics Library](https://github.com/oneapi-src/oneDAL)
 - [About Intel(R) daal4py](https://github.com/intel/scikit-learn-intelex/tree/master/daal4py)
+- [Jupyter examples](https://github.com/intel/scikit-learn-intelex/tree/master/examples/notebooks)
```
Comment (PetrovKP) [inline README.md:37]: Move to first

File: examples/notebooks/README.md [hunk 5]
```diff
@@ -1,12 +1,22 @@
 # Intel(R) Extension for Scikit-learn* notebooks :snake:
```
Comment (PetrovKP) [inline examples/notebooks/README.md:1]:
```suggestion
#  :snake: Intel(R) Extension for Scikit-learn* notebooks
```

File: examples/notebooks/README.md [hunk 6]
```diff
@@ -1,12 +1,22 @@
...
+|    Logistic Regression  |   CIFAR-100  | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb)    | [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) |
+|          SVC            |     Adult    | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/svc_adult.ipynb) | [sklearn.svm.SVC](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html) |
+| K nearest neighbors classifier | MNIST | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/knn_mnist.ipynb) |    [sklearn.neighbors.KNeighborsClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html) |
+|        Nu SVR           | Medical charges | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/nusvr_medical_charges.ipynb) | [sklearn.svm.NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html) |
```
Comment (PetrovKP) [inline examples/notebooks/README.md:21]:
```suggestion
|        NuSVR           | Medical charges | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/nusvr_medical_charges.ipynb) | [sklearn.svm.NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html) |
```

File: examples/notebooks/README.md [hunk 7]
```diff
@@ -1,12 +1,22 @@
...
 ```bash
-pip install notebook
+pip install notebook scikit-learn-intelex
 ```  
+
+#### :pencil: Table of contents
+
+| Algorithm               | Workload       | Notebook       | Sckit-learn estimator|
+| :----------------------:| :------------: | :-------------:| :-------------------:|
+|    Logistic Regression  |   CIFAR-100  | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb)    | [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) |
```
Comment (PetrovKP) [inline examples/notebooks/README.md:18]:
```suggestion
|    LogisticRegression  |   CIFAR-100  | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb)    | [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) |
```

Comment (PetrovKP) [reply examples/notebooks/README.md:18]: and other - I think better merge

File: README.md (hunk 4 above)
Comment (PetrovKP) [reply README.md:37]: not applied

Comment (SmirnovEgorRu) [reply README.md:37]: To be honest - they are not jupyter notebooks.  jupyter - just a way how to work with .ipynb.
So, better to say "Python notebook examples" or "Notebook examples"

File: examples/notebooks/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+# :snake: Intel(R) Extension for Scikit-learn* notebooks
+
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.
```
Comment (alexandraepan) [inline examples/notebooks/README.md:3]:
```suggestion
This folder contains examples of Jupyter* notebooks that use Intel(R) extension for Scikit-learn for popular datasets.  
```

File: examples/notebooks/README.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# :snake: Intel(R) Extension for Scikit-learn* notebooks
+
+This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.  
+You need jupyter notebook to run these .ipynb files:
```
Comment (alexandraepan) [inline examples/notebooks/README.md:4]:
```suggestion
You need Jupyter* notebook to run the following .ipynb files:
```

File: README.md (hunk 4 above)
Comment (alexandraepan) [inline README.md:37]:
```suggestion
- [Jupyter* examples](https://github.com/intel/scikit-learn-intelex/tree/master/examples/notebooks)
```

Comment (Pahandrovich) [reply README.md]: Thanks, fixed

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**